pkg/web/web.go

```diff
@@ -188,6 +190,7 @@ func New(ctx context.Context, opts ...Option) (*Server, error) {
 		mux.MiddlewareFunc(webmiddleware.Log(logger, options.logIgnorePaths)),
 		mux.MiddlewareFunc(webmiddleware.Cookies(hashKey, blockKey)),
 		mux.MiddlewareFunc(webmiddleware.NoCache),
+		otelmux.Middleware("ttn-lw-stack", otelmux.WithTracerProvider(tracing.FromContext(ctx))),
```

Comment:
Is it intentional that this is the last middleware?

The other middlewares are, or can be, request specific, so I don't see a reason why they would not be included in the trace.

Reply:
It wasn't intentional, moved it to the top spot.

Reply:
It should be below Recover, because if this middleware panics, TTS panics.

Is there anything in the tracer that comes from the request context filled by TTS? In that case, it should also below `FillContext`. But it shouldn't be (much) lower.

Reply:
> Is there anything in the tracer that comes from the request context filled by TTS?

No, then just below panic middleware.